### PR TITLE
App ported to Lab4

### PR DESCRIPTION
**Pull request by SameerK-MSFT**
Fixes # 
Sampel App ported to Lab4

**Changes proposed in this request**
Updated the Lab IDs

**Testing**
Manual

**Performance impact**
None

### CODE REVIEW

**Review by bgavrilMS — APPROVED** [review]

File: tests/devapps/Intune-xamarin-Android/PCAWrapper.cs [hunk 1]
```diff
@@ -35,13 +35,13 @@ public class PCAWrapper
         private const string _authority = "https://login.microsoftonline.com/organizations";
 
         // ClientID of the application
-        private static string _clientID = "6d50af5d-2529-4ff4-912f-c1d6ad06953e"; // TODO - Replace with your client Id. And also replace in the AndroidManifest.xml
+        private static string _clientID = "bd9933c9-a825-4f9a-82a0-bbf23c9049fd"; // TODO - Replace with your client Id. And also replace in the AndroidManifest.xml
 
         // redirect URI for the application
         private static string _redirectURI = $"msauth://com.sameerk.intune.test.xamarin/EHyvOdXj4uLXJXDaOMy5lwANmp0="; // TODO - Replace with your redirectURI
 
         // TenantID of your organization
```
Comment (gladjohn) [inline tests/devapps/Intune-xamarin-Android/PCAWrapper.cs:43]:
```suggestion
        // TenantID of your organization (msidlab4.com)

```

**Review by gladjohn — APPROVED** [review]
looks good Sameer, just a few comments to add lab details for future reference

File: tests/devapps/Intune-xamarin-Android/PCAWrapper.cs [hunk 2]
```diff
@@ -35,13 +35,13 @@ public class PCAWrapper
         private const string _authority = "https://login.microsoftonline.com/organizations";
 
         // ClientID of the application
```
[the remark this answers is not in the document]
Comment (SameerK-MSFT) [reply tests/devapps/Intune-xamarin-Android/PCAWrapper.cs]: Done

File: tests/devapps/Intune-xamarin-Android/PCAWrapper.cs (hunk 1 above)
Comment (SameerK-MSFT) [reply tests/devapps/Intune-xamarin-Android/PCAWrapper.cs]: Done

**Pull request merged**